### PR TITLE
Rashes amends callout

### PR DESCRIPTION
**Pull request by bottlerocket**

### CODE REVIEW

File: views/symptoms/rashes_meningitus.html [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+            </div>
+
+            <!--  Start main content -->
+            <p>The information below shouldn’t be used to self-diagnose your child’s condition, but it should give you a better idea of what is causing the rash. </p>
+
+            <h2 class="heading-xlarge" id="fever">Rash with fever</h2>
+
+            <div class="callout emergency" id="callout-999">
+
+                <h3>Meningitus</h3>
```
Comment (Demotive) [inline views/symptoms/rashes_meningitus.html]: Spelling!

File: views/symptoms/rashes_meningitus.html [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+                <p>If your child seems unwell with a rash and has a fever call 111 or speak to your GP.</p>
+
+            </div>
+
+            <!--  Start main content -->
+            <p>The information below shouldn’t be used to self-diagnose your child’s condition, but it should give you a better idea of what is causing the rash. </p>
+
+            <h2 class="heading-xlarge" id="fever">Rash with fever</h2>
+
+            <div class="callout emergency" id="callout-999">
```
Comment (teneightfive) [inline views/symptoms/rashes_meningitus.html]: There's 2 `div`'s with the id `callout-999`. This line and line 29. 

If it's there for an anchor link maybe just choose 1, or use different IDs.

File: views/symptoms/rashes_meningitus.html [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+                <div class="image-description">
+                    <p><a href="http://www.nhs.uk/conditions/chickenpox/Pages/Introduction.aspx">Chickenpox</a> causes red spots that turn to blisters.  They can be itchy.  They eventually scab and fall off.  Some children have a few spots, while others have them all over their body.</p>
+                </div>
+
+            </div>
+
+            <!-- Blisters on hands and feet -->
+            <div class="image-module">
+
+                <h3>Blisters on hands and feet and in the mouth. </h3>
```
Comment (teneightfive) [inline views/symptoms/rashes_meningitus.html]: Seems like an accidental period in this heading.

Comment (teneightfive) [inline views/symptoms/rashes_meningitus.html]: Might need to update the other rashes page too

File: views/symptoms/rashes_meningitus.html [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+            <!-- Pink red rash-->
+            <div class = "image-module">
+                
+                <h3>Pink, red rash</h3>
+
+                <img src="/images/rashes/rashes-scarlet-fever.jpg" alt="Image of scarlet fever rash" />
+
+                <div class="image-description">
+
+                    <p><a href="http://www.nhs.uk/Conditions/Scarlet-fever/Pages/Introduction.aspx">Scarlet fever </a> causes a pink-red rash, which feels like sandpaper and looks like sunburn.</p>
```
Comment (teneightfive) [inline views/symptoms/rashes_meningitus.html]: Remove the space after the anchor so the underline doesn't extend past the word

File: views/symptoms/rashes_meningitus.html [hunk 5]
```diff
@@ -0,0 +1,316 @@
...
+            </div>
+
+          </div>
+
+      </div> <!-- end row -->
+      
+  </div>  
+  
+</div>
+
```
Comment (teneightfive) [inline views/symptoms/rashes_meningitus.html:303]: Maybe drop some of this whitespace if you're making other changes to this PR

**Pull request merged**